radio/src/gui/128x64/model_setup.cpp

```diff
@@ -1230,6 +1232,16 @@ void menuModelSetup(event_t event)
         break;
       }
 
+#if defined(HARDWARE_INTERNAL_MODULE)
+      case ITEM_MODEL_SETUP_INTERNAL_MODULE_RACING_MODE:
+      {
+        ModuleData & moduleData = g_model.moduleData[moduleIdx];
+        lcdDrawTextAlignedLeft(y, STR_RACING_MODE);
+        moduleState[moduleIdx].racingMode = editCheckBox(moduleState[moduleIdx].racingMode, MODEL_SETUP_2ND_COLUMN, y, "", attr, event);
```

Comment:
it won't be saved in the model storage, it means that on watchdog reset we will be in an unknown state